### PR TITLE
updating tests

### PR DESCRIPTION
**Pull request by iruzevic**

### CODE REVIEW

**Review by dingo-d — APPROVED** [review]

File: tests/data/src/Blocks/components/typography/manifest.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+				"label": "14 Text Roman",
+				"value": "14-text-roman"
+			},
+			{
+				"label": "12 Text Roman",
+				"value": "12-text-roman"
+			}
+		],
+		"typographyColor": [
+			"infinum",
```
Comment (goranalkovic-infinum) [inline tests/data/src/Blocks/components/typography/manifest.json:125]: Isn't this a bit too project-specific? 😄

Comment (dingo-d) [reply tests/data/src/Blocks/components/typography/manifest.json:125]: Yeah, it should be eightshift :trollface:

Comment (dingo-d) [reply tests/data/src/Blocks/components/typography/manifest.json:125]: All jokes aside, this is a test mock data, doesn't really matter what it says here 😄

**Review by gabriel-glo — APPROVED** [review]

File: tests/data/src/Blocks/components/typography/manifest.json (hunk 1 above)
Comment (iruzevic) [reply tests/data/src/Blocks/components/typography/manifest.json:125]: yea I just copied from the project so no mater.

**Pull request merged**